src/Components/RecsListTable/RecsListTable.spec.ct.js

```diff
@@ -783,9 +797,11 @@ describe('empty recommendations list table', () => {
   });
 
   it('renders error message', () => {
-    cy.get('#error-state-message')
-      .find('h4')
-      .should('have.text', 'Something went wrong');
+    checkEmptyState(
```

Comment:
This is not empty state, but  a message, right? Should we rename the method to `checkMessage`?

Reply:
So in this test, the `something wen wrong` is displayed withing an EmptyState object?

Reply:
ok